src/dp_influx.erl

```diff
@@ -52,10 +51,19 @@ parse_tag(<<"=", V/binary>>, K) ->
 parse_tag(<<C, R/binary>>, K) ->
     parse_tag(R, <<K/binary, C>>).
 
+%% Time format values such as \" 0:00 \" are ignored
+parse_metrics(<<"\" ", _F:4/binary, "\"", R/binary>>, _Metric, Metrics, M) ->
+    parse_metrics(R, <<>>, Metrics, M);
```

Comment:
that doesn't seem like a good approach, if influx sends strings they might be more then a space and 4 characters, or even be escaped. I fear the real approach here is to pars strings and ignore them all togehter